imports/plugins/custom/wallet/client/wallet.js

```diff
@@ -0,0 +1,218 @@
+import { Meteor } from "meteor/meteor";
+import { Template } from "meteor/templating";
+import { Reaction } from "/client/api";
+import { Accounts, Packages, Wallets } from "/lib/collections";
+import "../lib/api/paystackApi";
+
+let list = [];
+let pageList = [];
+let currentPage = 1;
+const numberPerPage = 10;
+
+Template.wallet.onCreated(function bodyOnCreated() {
+  this.state = new ReactiveDict();
+  this.state.setDefault({
+    details: { balance: 0, transactions: [] },
+    transactions: [],
+    transactionsList: [],
+    currentPageNum: 1
+  });
+  this.autorun(() => {
+    this.subscribe("transactionDetails", Meteor.userId());
+    const transactionInfo = Wallets.find().fetch();
+    if (transactionInfo.length > 0) {
+      this.state.set("details", transactionInfo[0]);
+      makeList();
+    }
+  });
+});
+
+function makeList() {
+  const transactions = Template.instance().state.get("details").transactions;
+  if (transactions.length > 0) {
+    const newTranscationDetails = transactions.sort(function (a, b) {
+      return new Date(b.date) - new Date(a.date);
+    });
+    Template.instance().state.set("transactionsList", newTranscationDetails);
+    numberOfPages = getNumberOfPages();
+  }
+}
+
+function getNumberOfPages() {
+  list = Template.instance().state.get("transactionsList");
+  if (list.length > 0) {
+    return Math.ceil(list.length / numberPerPage);
+  }
+}
+
+function check() {
+  document.getElementById("next").disabled = currentPage === numberOfPages
```

Comment:
no need to use ternary operator here, just do
`document.getElementById("previous").disabled = currentPage === 1` and so on
why? logical comparison returns boolean values